waspc/src/Wasp/AppSpec/Valid.hs

```diff
@@ -379,6 +396,42 @@ validateWebAppBaseDir spec = case maybeBaseDir of
     startsWithSlash ('/' : _) = True
     startsWithSlash _ = False
 
+validateUserNodeVersionRange :: AppSpec -> [ValidationError]
+validateUserNodeVersionRange spec =
+  concat
+    [ checkUserRangeIsInWaspRange,
+      checkUserRangeAllowsOnlyMinorChanges
+    ]
+  where
+    userRange = AS.userNodeVersionRange spec
+    userVersionInterval = SVB.versionBounds userRange
+    waspVersionInterval = SVB.versionBounds $ SV.backwardsCompatibleWith oldestWaspSupportedNodeVersion
+    checkUserRangeIsInWaspRange :: [ValidationError]
+    checkUserRangeIsInWaspRange =
+      if not (userVersionInterval `SVB.isSubintervalOf` waspVersionInterval)
+        then
+          [ GenericValidationError $
+              "Node version range that you specified for your Wasp app ("
+                <> show userRange
+                <> ") has too low lower boundary. It must be >= "
```

Comment:
I think there's something off with this sentence. Can we do something like:
```
Your app's Node version range (`range`) allows versions lower than `oldestWaspSupportedNodeVersion`.
Wasp only works with Node >= version`.
```

Reply:
Thanks this is a nice suggestion, done!